third_party/ienn/src/ie_compilation.cpp

```diff
@@ -156,28 +152,28 @@ int32_t Compilation::AddInput(uint32_t index) {
     return result;
   }
   try {
-    std::string name(std::to_string(index));
-    size_t layer_id =
-        builder_->addLayer(Builder::InputLayer(name).setPort(Port(dims)));
-    layer_id_map_[index] = layer_id;
+    auto input_node =
+        std::make_shared<op::Parameter>(element::f32, Shape(dims));
+    index_op_map_[index] = input_node;
```

Comment:
Meger the two lines.
`index_op_map_[index] = std::make_shared<op::Parameter>(element::f32, Shape(dims));`